admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -51,6 +51,7 @@ docker run --detach=true \
     -e "QSERV_MASTER=$MASTER" \
     $DATA_VOLUME_OPT \
     $LOG_VOLUME_OPT \
+    -v /etc/localtime:/etc/localtime:ro \
```

Comment:
The preference is for all machine activities to be in UTC, but if that is not feasible, local timezone is allowed.  All logs must have the timezone in each timestamp.  See DM-1203.